docs/cli-reference/dfx-envars.mdx

```diff
@@ -8,17 +8,19 @@ You can configure certain properties for your SDK execution environment using en
 
 This section lists the environment variables that are currently supported with examples of how to use them. In most cases, you can set environment variables for a session by executing an command in the terminal or by adding a line similar to the following to your `.profile` file:
 
-    export DFX_NETWORK=ic
+```shell
+export DFX_NETWORK=ic
```

Comment:
Just noticing this now, but this is a terrible example, because dfx doesn't look at DFX_NETWORK when determining which network to connect to.  I've created https://dfinity.atlassian.net/browse/SDKTG-313 to track this